client/components/ReviewRating/ReviewList/StarRating.jsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import { AiTwotoneStar, AiOutlineStar } from 'react-icons/ai';
+
+const StarRating = () => (
```

Comment:
Good chance for your teammates to reuse this component once it's complete! I believe other sections will also use this stars feature

Reply:
I push this part accidentally, just getting the idea of how to create stars.